### DISCUSSION TITLE
Import: Preselect datasource with the same name

### DISCUSSION THREAD

**Discussion by fatbasstard** [grafana/grafana#71532] [Dashboards, Panels & Visualizations]
Hi,

I'm playing around with export/import with the "Export for sharing externally" so I can copy dashboards between environment (e.g. production dashboard to development to test a new Grafana or plugin version). This option allows me to "fix" the datasources (since the `UUID` is exported and not the name).

The datasource I use exists in both instances of Grafana with the same name, just with a different UUID, when importing I get the option to map the datasources:

![image](https://github.com/grafana/grafana/assets/35613489/b85ccafc-bdea-44b5-8997-8449d669bfaf)

It would be great/nice if it suggests/preselects the datasource if it exists with the same name.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82606. Feel free to continue the discussion around this there. Thank you!

**Reply by fatbasstard** [reply]
Love it, since it was an issue first 🤣
